assets/js/main.js

```diff
@@ -0,0 +1,308 @@
+var taskName = document.getElementById("taskName");
+var assignee = document.getElementById("assignee");
+
+var addTaskButton = document.getElementById("addTaskButton");
+var clearTaskButton = document.getElementById("clearTaskButton");
+
+var tasksData = document.getElementById("tasksData");
+var tasksDataDone = document.getElementById("tasksDataDone");
+
+var tasksArray = [];
+
+var deleteIcon = document.querySelector(".deleteIcon");
+var deleteAllButton = document.querySelector("#deleteAllButton");
+
+var currentIndex;
+
+var countTask = document.querySelector("#countTask");
+var countDone = document.querySelector("#countDone");
+
+var toggledBtn = document.querySelector("#toggledBtn");
+var toggle = 0;
+
+var nameAlert = document.getElementById("nameAlert");
+
+// to check if the local storage empty or not
+if (localStorage.getItem("tasksList") == null) {
+  tasksArray = [];
+} else {
+  tasksArray = JSON.parse(localStorage.getItem("tasksList"));
+  displayData();
+  counter();
```

Comment:
Its better if you can group those two into another function that has a proper name and then call that function here.

```suggestion
 initializeData()
```

`const initializeData = () =>{
  displayData();
  counter();
}`